css/free5e.scss

```diff
@@ -26,3 +26,12 @@ table {
   table-layout: auto;
   width: 100%;
 }
+
+.spell {
+  font-style: italic;
+  color: #152347;
+
+  &:after {
+    content: '\00a0🪄';
+  }
+}
```

Comment:
This is a neat little thing we can do given the new changes. For the HTML output (and the HTML output only) we now get this:

<img width="972" height="95" alt="CSS highlighting of spells in the HTML output, adding a magic wand emoji after each spell" src="https://github.com/user-attachments/assets/30565cea-be9c-4bc3-bb93-ee52ed31f9a2" />

And since we're adding the wand emojis via CSS, screenreaders ignore them. 🙂